### PR TITLE
Llu/matmul replace compute at

### PR DESCRIPTION
**Pull request by liqiangxl**
replace compute at in matmul with inlinemost.
no infulence of performance, all tests passed.

### CODE REVIEW

**Review by zasdfgbnm — APPROVED** [review]
Thanks for cleaning up!

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

File: csrc/scheduler/matmul.cpp [hunk 1]
```diff
@@ -717,6 +701,12 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
       {acr, bcr, ab, bb, a, b},
       {ParallelType::TIDy, ParallelType::TIDz});
 
+  // auto inline for all tensors except register tensors and output tensor
+  inlineMost(ir_utils::allTvsExcept(fusion, {acr, bcr, ab, bb, c}));
+
+  // if auto inline, will inline to position-7, leads to performance regression
+  inlineSelectedAt({acr, bcr, ab, bb}, {cc}, 6);
```
Comment (crcrpar) [inline csrc/scheduler/matmul.cpp:708]: novice question: why would `cc` need curly brackets?

Comment (zasdfgbnm) [reply csrc/scheduler/matmul.cpp:708]: No, it is not needed. Thanks for reporting. I will write a quick fix right now.

Comment (crcrpar) [reply csrc/scheduler/matmul.cpp:708]: for the record https://github.com/NVIDIA/Fuser/pull/205

**Pull request merged**